client/lib/jetpack/paths.ts

```diff
@@ -41,6 +41,8 @@ export const partnerPortalBasePath = ( path = '' ) => `/partner-portal${ path }`
 export const agencySignupBasePath = () => '/manage/signup';
 export const agencySignupLegacyPath = () => '/agency/signup';
 
+export const overviewPath = () => '/overview';
```

Comment:
Not a blocker, but I'm curious if there's a technical reason (other than "that's what we do elsewhere") for we hardcode the path twice instead of setting the constant once (`JETPACK_MANAGE_OVERVIEW_LINK`) and using that going forward...

Reply:
AFAIK then no. It's just a natural progression that has happened over time ⏲️